src/txmempool.cpp

```diff
@@ -50,8 +51,23 @@ CTxMemPoolEntry::CTxMemPoolEntry(const CTxMemPoolEntry& other)
     *this = other;
 }
 
-double
-CTxMemPoolEntry::GetPriority(unsigned int currentHeight) const
+bool CTxMemPoolEntry::IsSuperStandardImmediate() const
```

Comment:
This would require passing a chain reference through the respend framework as well. I think it's fine to rely on the mempool to be aware of the chain (the mempool is also intimately aware of the UTXO set).